apps/docs/content/docs/components/select.mdx

```diff
@@ -148,6 +148,12 @@ You can combine the `isInvalid` and `errorMessage` properties to show an invalid
 
 <CodeDemo title="With Error Message" files={selectContent.errorMessage} />
 
+### Clear Button
+
+If you pass the `isClearable` property to the select, it will have a clear button which will be visible only when some value is selected.
```

Comment:
```diff
-it will have a clear button which will be visible only when some value is selected.
+it will have a clear button which will be visible only when a value is selected.
```